src/content/developers/docs/spin-up-your-node/index.md

```diff
@@ -0,0 +1,164 @@
+- Manually calling them with a suitable protocol (e.g. using `curl`)
+- Attaching a provided console (e.g. `geth attach`)
+- Implementing them in applications 
+
+Different clients have different implementations of the RPC endpoints. But there is a standard JSON-RPC which you can use with every client. For an overview [read the JSON-RPC docs](https://eth.wiki/json-rpc/API). 
+Applications that need information from the Ethereum network can use this RPC. For example, popular wallet MetaMask lets you [run a local blockchain instance and connect to it](https://metamask.zendesk.com/hc/en-us/articles/360015290012-Using-a-Local-Node). 
+
+#### Reaching RPC {#reaching-rpc}
+
+The default port of JSON-RPC is `8545` but you can modify the ports of local endpoints in the config file. By default, the RPC interface is only reachable on the localhost of your computer. To make it remotely accessible, you might want to expose it to the public by changing the address to `0.0.0.0`. This will make it reachable over local and public IP addresses. In most cases you'll also need to set up port forwarding on your router.
+
+You should do this with caution as this will let anyone on the internet control your node. Malicious actors could access your node to bring down your system or steal your funds if you're using your client as a wallet.
+
+A way around this is to prevent potentially harmful RPC methods from being modifiable. For example, with `geth`, you can declare modifiable methods with a flag: `--http.api web3,eth,txpool`.
+
+You can also host access to your RPC interface by pointing service of web server, like Nginx, to your client's local address and port.
+
+The most privacy-preserving and simple way to set up a publicly reachable endpoint, you can host it on your own [Tor](https://www.torproject.org/) onion service. This will let you reach the RPC outside your local network without a static public IP address or opened ports. 
+To do this:
+- Install `tor`
+- Edit `torrc` config to enable hidden service with address of your client's RPC address and port
+- Restart `tor` service
+
+Once you restart Tor, you'll get hidden service keys and a hostname in your desired directory. From then, your RPC will be reachable on a `.onion` hostname.
+
+### Operating the node {#operating-the-node}
+
+You should regularly monitor your node to make sure it's running properly. You may need to do occasional maintenance.
+
+#### Keeping node online {#keeping-node-online}
+
+Your node doesn't have to be online nonstop but you should keep it online as much as possible to keep it in sync with the network. You can shut it down to restart it but keep in mind that:
+- Shutting down can take up to a few minutes if the recent state is still being written on disk.
+- Forced shut downs can damage the database.
+- Your client will go out of sync with the network and will need to resync when you restart it.
```

Comment:
Perhaps just make a small disclaimer for anyone thinking about running a node for STAKING purposes that they should absolutely try to minimize downtime.

Reply:
Thanks for feedback, adding disclaimer.

```suggestion
#### Keeping node online {#keeping-node-online}

Your node doesn't have to be online nonstop but you should keep it online as much as possible to keep it in sync with the network. You can shut it down to restart it but keep in mind that:
- Shutting down can take up to a few minutes if the recent state is still being written on disk.
- Forced shut downs can damage the database.
- Your client will go out of sync with the network and will need to resync when you restart it.

*This doesn't aply on Eth2 validator nodes.* Taking your node offline will affect all services dependent on it. If you are running a node for *staking* purposes you should try to minimize downtime as much as possible.
```